cmd/cli/job/executions.go

```diff
@@ -148,10 +148,10 @@ func (o *ExecutionOptions) run(cmd *cobra.Command, args []string, api client.API
 	response, err := api.Jobs().Executions(ctx, &apimodels.ListJobExecutionsRequest{
 		JobID: jobID,
 		BaseListRequest: apimodels.BaseListRequest{
-			Limit:     o.Limit,
-			NextToken: o.NextToken,
-			OrderBy:   o.OrderBy,
-			Reverse:   o.Reverse,
+			Limit:     o.ListOptions.Limit,
+			NextToken: o.ListOptions.NextToken,
+			OrderBy:   o.ListOptions.OrderBy,
+			Reverse:   o.ListOptions.Reverse,
```

Comment:
hmm .. how do we feel about enforcing this rule? I am okay with disabling the rule in .golangci.yml if you think we are better off without it

Reply:
@wdbaruni I tend to lean toward verbosity here, and in general, which is more of a personal choice.
But I'll add the exception to the definitions file to balance things out.

Reply:
Change reverted